src/wazuh_db/schema_agents.sql

```diff
@@ -345,14 +347,19 @@ CREATE TABLE IF NOT EXISTS vuln_cves (
     version TEXT,
     architecture TEXT,
     cve TEXT,
-    PRIMARY KEY (name, version, architecture, cve)
+    reference TEXT DEFAULT '' NOT NULL,
+    type TEXT DEFAULT 'UNDEFINED' NOT NULL CHECK (type IN ('OS', 'PACKAGE','UNDEFINED')),
```

Comment:
Do we have a possibility in which the type could be `UNDEFINED`? I think that this type is not necessary.

Reply:
Sorry, now I understand the purpose of this. Considering the fact that it can't be `NULL`, you are defining a default status.
But I'm still wondering whether this is necessary. I understand that for the upgrades you need to set a value but. Do we have to keep the data in the table during the upgrades? I think that probably the best idea should be run a `BASELINE` scan.

Reply:
That status was created only to allow the correct CVEs insertion in the branch until the methods are modified. If there isn't a default value and the column can't be `NULL`, then no vulnerabilities could be inserted.

Setting the default value as **OS** or **PACKAGE** without an **UNDEFINED** state would have the same result.

Reply:
Good! So what we will do is to leave a comment on the issue in which we will implement the `insert` command to remove the `undefined` type as part of that change.